### PR TITLE
interfaces/builtin/network_setup_control: allow busctl to bind

### PR DESCRIPTION
**Pull request by bboozzoo**
Starting from systemd 254, busctl will attempt to bind the socket to a known path to facilitate debugging. The change was introduced in https://github.com/systemd/systemd/commit/a0cb33581630a54c89d088d36eb3bf6cf7459cd7 and the path is composed of `@<hex-encoded-u64>/bus/<command>/<description>`.

Actual denial:
```
Feb 29 08:22:04 localhost kernel: audit: type=1400 audit(1709194924.547:213):
apparmor="DENIED" operation="bind" class="net" profile="snap.console-conf.console-conf"
pid=4504 comm="busctl" family="unix" sock_type="stream" protocol=0
requested_mask="bind" denied_mask="bind" addr="@f46ecfc1c63ae80f/bus/busctl/busctl"
```

### CODE REVIEW

File: interfaces/builtin/network_setup_control.go [hunk 1]
```diff
@@ -45,6 +45,10 @@ const networkSetupControlConnectedPlugAppArmor = `
 # Netplan uses busctl internally, so allow using that as well
 /usr/bin/busctl ixr,
 
+# from systemd 254 onward, busctl binds to a unix socket upon startup to
+# facilitate debugging
+unix (bind) type=stream addr="@[0-9a-fA-F]*/bus/busctl/*"
```
Comment (zyga) [inline interfaces/builtin/network_setup_control.go:50]: I think this is fine but I suspect we may want to have something like it in more places.

**Review by zyga — APPROVED** [review]

File: interfaces/builtin/network_setup_control.go (hunk 1 above)
Comment (bboozzoo) [reply interfaces/builtin/network_setup_control.go:50]: aah, yes, good point, let me push one more patch for network-setup-observe

Comment (bboozzoo) [reply interfaces/builtin/network_setup_control.go]: also missed a `,` at the end here

**Review by alexmurray — COMMENTED** [review]
LGTM (FWIW even though this was already merged)

**Pull request merged**